lib/help_scout/api/access_token.rb

```diff
@@ -1,10 +1,40 @@
 # frozen_string_literal: true
 
+require 'date'
```

Comment:
It initially struck me as weird that we had to require "date" but not "time". I forget `Time` is part of Ruby Core Lib and `Date` is part of the Ruby Std Lib.

Reply:
Yeah that gets me every *time* 🥁 (and I normally try and require both `datetime` and `date_time` first!).